### PR TITLE
use networking.k8s.io/v1 version of Ingress for service exposure guide in Rancher

### PR DESCRIPTION
**Pull request by sr229**
Thank you for wanting to submit a Pull Request to the Civo Kubernetes Marketplace repository!

If your pull request is to submit a new application to the marketplace, please answer the following questions:

* [ ] Have you followed the guidelines in our Contributing document in setting up the application?
* [ ] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [ ] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
* [ ] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?

If your pull request concerns an existing Marketplace application, please make sure you have:

* [x] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.
* [x] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).
* [x] Tested that the application works with the proposed updates applied (including a screenshot)

### CODE REVIEW

File: rancher/post_install.md [hunk 1]
```diff
@@ -15,13 +14,18 @@ metadata:
     kubernetes.io/ingress.class: traefik
 spec:
   rules:
-  - http:
+  - host: example.com
```
Comment (saiyam1814) [inline rancher/post_install.md:17]: Can you test with host : rancher.<your-cluster-id>.k8s.civo.com ? if that works please change it

**Review by saiyam1814 — CHANGES_REQUESTED** [review]
Can you test with host : rancher..k8s.civo.com ? if that works please change it

**Comment by saiyam1814** [comment]
this was first initiated in #78, can you both work this out and see if  one of the PR can be closed ?

**Comment by sr229** [comment]
> this was first initiated in #78, can you both work this out and see if one of the PR can be closed ?

Looks like my version is much more updated than his own so I believe GH-78 should be closed.

File: rancher/post_install.md (hunk 1 above)
Comment (sr229) [reply rancher/post_install.md:17]: Tested it even with other dynamic DNS providers and I can confirm this ingress configuration should just work (both Traefik and NGINX).

**Comment by saiyam1814** [comment]
Cool, Can you check that host thing ?

**Comment by sr229** [comment]
> Cool, Can you check that host thing ?

Works as intended (though the Node crashed since the test node is way too small to even run Rancher D:)

**Comment by saiyam1814** [comment]
@sr229 I am trying to make the ingress same across apps like the portainer, kubevious one , can we have something like that ?

**Comment by sr229** [comment]
> @sr229 I am trying to make the ingress same across apps like the portainer, kubevious one , can we have something like that ?

If you mean a common ingress configuration, as long as they have the same endpoint, port and path they should just work.

**Comment by saiyam1814** [comment]
by common I mean like the host should be rancher.clusterid..k8s.civo.com

**Comment by sr229** [comment]
> by common I mean like the host should be rancher.clusterid..k8s.civo.com

Configuration should just work, if you have some sort of templating engine like what Helm does to replace placeholder variables then you can.

**Comment by saiyam1814** [comment]
ok I will try to explain again 
take portainer example
```
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: portainer
  namespace: portainer
spec:
  rules:
  - host: portainer.<your-cluster-id>.k8s.civo.com
    http:
      paths:
      - backend:
          serviceName: portainer
          servicePort: 9000
```

So I want a similar post install steps for Rancher as well.

**Comment by sr229** [comment]
> ok I will try to explain again
> take portainer example
> 
> ```
> apiVersion: extensions/v1beta1
> kind: Ingress
> metadata:
>   name: portainer
>   namespace: portainer
> spec:
>   rules:
>   - host: portainer.<your-cluster-id>.k8s.civo.com
>     http:
>       paths:
>       - backend:
>           serviceName: portainer
>           servicePort: 9000
> ```
> 
> So I want a similar post install steps for Rancher as well.

If you mean having them use the cluster DNS, then yes it would work, since we're using Civo-specific dynamic DNS, you can use `*.<clusterid>.k8s.civo.com`

**Comment by saiyam1814** [comment]
Yup, so can you change the post install to look similar to above ?

**Comment by saiyam1814** [comment]
lgtm

**Pull request merged**